contrib/monitored_resource_util/src/main/java/io/opencensus/contrib/monitoredresource/util/MonitoredResource.java

```diff
@@ -87,23 +69,29 @@ public ResourceType getResourceType() {
      * @return the AWS account ID.
      * @since 0.13
      */
-    public abstract String getAccount();
+    public String getAccount() {
+      return firstNonNull(awsInstanceLabels.get(AwsEc2InstanceResource.ACCOUNT_ID_KEY), "");
```

Comment:
`firstNonNull here is not needed because it's already called: https://github.com/census-instrumentation/opencensus-java/pull/1718/files#diff-e0f6d1e015f871479d844cca09da8bc7R34. Same for all others in this class.

Reply:
Ignore this comment - just saw this is the key not the value.